pkg/provider/identity.go

```diff
@@ -82,21 +82,23 @@ type ServerConfig struct {
 type ServerOptions struct {
 	Config   ServerConfig
 	Ident    *FullIdentity
+	RevDB    *peertls.RevocationDB
```

Comment:
This looks very weird to close "Options".

Reply:
do you have an alternate name or flow in mind?

Reply:
not at the moment